devicetypes/smartthings/smartweather-station-tile.src/capability.smartWeather.json

```diff
@@ -0,0 +1,244 @@
+{
+  "name": "Smart Weather",
+  "attributes": {
+    "localSunrise": {
+      "schema": {
+        "type": "object",
+        "properties": {
+          "value": {
+            "type": "string"
+          }
+        },
+        "additionalProperties": false
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "localSunset": {
+      "schema": {
+        "type": "object",
+        "properties": {
+          "value": {
+            "type": "string"
+          }
+        },
+        "additionalProperties": false
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "localSunrise": {
+      "schema": {
+        "type": "object",
+        "properties": {
+          "value": {
+            "type": "string"
+          }
+        },
+        "additionalProperties": false
+      },
+      "required": [
+        "value"
+      ]
+    },
```

Comment:
Very noobish when it comes to these files, but why do we list `localSunrise` twice in the attributes list?

Reply:
Uhhhhhhh $20 sits on a copy and paste error. :P